bndtools.builder/_plugin.xml

```diff
@@ -36,8 +43,49 @@
 			<description>Marks exported packages with a '+' decoration, and
 				excluded packages with a strike-through.</description>
 		</decorator>
-	</extension>
 
+		<decorator
+			id="bndtools.componentDecorator"
+			label="Bndtools OSGi Component Decorations"
+			state="true"
+			adaptable="true"
+			lightweight="true"
+			class="org.bndtools.builder.decorator.ui.ComponentDecorator"
+			location="TOP_LEFT">
```

Comment:
Quick question, so for the package the decoration is TOP_RIGHT but for component it is TOP_LEFT.  How about with use TOP_RIGHT for both?  what do you think?